### ISSUE TITLE
Cast values on update

### ISSUE THREAD

**Issue by jonataswalker** [citycide/trilogy#31]
Shouldn't we use this on update:

```javascript
// this is from `create()`
let insertion = types.toDefinition(this, object)
```

I have an error trying to update with a boolean value.

**Comment by citycide** [comment]
@jonataswalker I've pushed a fix - are you able to pull & test to see if it fixes your problem before I publish? I did add a test that works.

**Comment by jonataswalker** [comment]
Fix confirmed @citycide. Thanks.

**Comment by citycide** [comment]
Released in [v1.0.0-rc.5](https://github.com/citycide/trilogy/releases/tag/v1.0.0-rc.5).
